tests/CMakeLists.txt

```diff
@@ -263,6 +263,24 @@ if(WITH_PYTHON)
     add_fdb_test(TEST_FILES fast/MockDDReadWrite.toml)
   endif()
 
+  if (NO_MULTIREGION_TEST)
```

Comment:
I am not sure about "IGNORE" is the right flag we should use here. IGNORE means, if RUN_IGNORED_TESTS is turned on, we still run these tests. I am thinking not including them in the test at all if they are not supported -- i.e. just skip these tests. However I have no background knowledge on these tests. So please ignore the comment if it is not the case.

Reply:
>If there are multiple factors impacting the final state of a test, it might be good to combine all the factors in the same place to avoid surprise in the future.

the call to configure_testing() takes a parameter of which tests to ignore, and this is done before calls to `add_fdb_test`.